update_conda_lock/check_prepare_env.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
```

Comment:
If we make this file Python it can work on windows / mac / linux? We already require Python for other parts.

Reply:
This script should work on all these OSs, there's nothing really OS-specific.